assets/app/scripts/controllers/settings.js

```diff
@@ -26,7 +26,31 @@ angular.module('openshiftConsole')
     ProjectsService
       .get($routeParams.project)
       .then(_.spread(function(project, context) {
-        $scope.project = project;
+        angular.extend($scope, {
+          project: project,
+          show: { display: true, form: false, actions: true },
```

Comment:
Yeah, I will eliminate.  It made sense when i had the edit icons, as i could return the regular view but keep the buttons hidden until the promise resolution.  Now that the 'edit' action is in the menu, no longer needed.